.github/workflows/analyze_at_app.yaml

```diff
@@ -0,0 +1,22 @@
+name: analyze at_app
+
+on:
+  push:
+    branches:
+      - trunk
+  pull_request:
+    branches:
+      - trunk
+
+jobs:
+  analyze_build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/setup_job
```

Comment:
This is my first time seeing this pattern of including a composite action, so it probably deserves a comment saying what's happening here.

(Initially I'm wondering how `dart analyze` can be used without installing Dart)?

We should probably be having a higher level conversation about establishing an actions repo with boilerplate we're likely to use across other repos.

Reply:
That's a good idea!

Reply:
Added the comment